### ISSUE TITLE
Fails with Gradle 4.3

### ISSUE THREAD

**Issue by pkubowicz** [tbroyer/gradle-apt-plugin#66]
```
Caused by: java.lang.NoClassDefFoundError: Could not initialize class net.ltgt.gradle.apt.CompatibilityUtils
	at net.ltgt.gradle.apt.AptPlugin$5.execute(AptPlugin.java:177)
	at net.ltgt.gradle.apt.AptPlugin$5.execute(AptPlugin.java:173)
	at org.gradle.internal.Actions$FilteredAction.execute(Actions.java:205)
	at org.gradle.api.internal.DefaultDomainObjectCollection.all(DefaultDomainObjectCollection.java:136)
	at org.gradle.api.internal.tasks.RealizableTaskCollection.all(RealizableTaskCollection.java:192)
	at org.gradle.api.internal.DefaultDomainObjectCollection.withType(DefaultDomainObjectCollection.java:159)
	at net.ltgt.gradle.apt.AptPlugin.configureCompileTasks(AptPlugin.java:169)
	at net.ltgt.gradle.apt.AptPlugin.apply(AptPlugin.java:33)
	at net.ltgt.gradle.apt.AptPlugin.apply(AptPlugin.java:30)
Caused by: java.lang.RuntimeException: java.lang.NoSuchMethodException: org.gradle.api.tasks.TaskPropertyBuilder.withPropertyName(java.lang.String)
        at net.ltgt.gradle.apt.CompatibilityUtils.getMethod(CompatibilityUtils.java:62)
        at net.ltgt.gradle.apt.CompatibilityUtils.<clinit>(CompatibilityUtils.java:34)
        ... 152 more
Caused by: java.lang.NoSuchMethodException: org.gradle.api.tasks.TaskPropertyBuilder.withPropertyName(java.lang.String)
        at net.ltgt.gradle.apt.CompatibilityUtils.getMethod(CompatibilityUtils.java:60)
        ... 153 more
```

**Comment by tbroyer** [comment]
Fixed in b403dd0; thanks for the heads up.

**Comment by tbroyer** [comment]
Just published v0.13 with the necessary changes (among other improvements)
